apps/src/templates/instructions/hint-prompt.module.scss

```diff
@@ -0,0 +1,6 @@
+.button {
+  border: 2px solid black;
+  &Yes {
+    margin-right: 5px;
```

Comment:
Just double checking if this also works in RTL?

Reply:
If it's not try `margin-inline-end: 5px;` — you could also update the margin here to `8px` (if it looks ok) to match the grid 🙌

Reply:
Great catch! The buttons were squished on prod in RTL so this little improvement is extra icing for RTL language users 🎉 

BEFORE on production 
![Screenshot 2024-08-22 at 11 36 43 AM](https://github.com/user-attachments/assets/a464b73e-0ff3-445d-858c-134f32edc530)

AFTER on local 
![Screenshot 2024-08-22 at 11 48 20 AM](https://github.com/user-attachments/assets/72fc99f6-17ab-4c0f-a8f5-70a4c2c1e675)